### ISSUE TITLE
Localization issues with dynamic terms

### ISSUE THREAD

**Issue by znarf** [opencollective/opencollective#2007]
**Describe the bug**

The way we have the translation string setup for collective/organization/individual is kind of broken. A simple solution might be to hardcode the 3 full strings instead of trying to construct them dynamically.

**Screenshot**

![Capture d’écran 2019-05-08 à 12 00 11](https://user-images.githubusercontent.com/806/57367260-f6b41a00-7188-11e9-9bc4-3ebdc7948ca0.png)

**Comment by Betree** [comment]
We'll have this problem in others parts of the site. We use the terms `This {collective}` very often so I'd like to come up with a generic solution, having a string that we can re-use for future implementations.

**Comment by Betree** [comment]
Hey @ykzts! Some feedback about Japanese would be really helpful here :slightly_smiling_face: 

If we have two translations like:

1. Id: `collective.thisType`
Default: `{type, select, COLLECTIVE {this collective} ORGANIZATION {this organization}}`
Example of expected result: `this organization`

2. Id: `collective.delete.label`
Default: `Delete {collective.thisType} now. It will be lost forever.`
Example of expected result: `Delete this organization now.`

Would that work for Japanese? Do you think handling strings this way will be future-proof or do you see cases where we're going to have problems?

**Comment by ykzts** [comment]
They can also be used correctly in Japanese. I will be happy if it is changed like that.

**Comment by Betree** [comment]
Thanks @ykzts! I'll do the change.

**Comment by znarf** [comment]
Unblocked!

**Comment by znarf** [comment]
@Betree is that still valid?

**Comment by Betree** [comment]
Yes, I've made some improvements on these but the exact problem referenced in this issue is still there

**Comment by Betree** [comment]
Got someone asking about this for Italian today.

@znarf A simple way to fix it would be to use the same term for all entries. Should we use the term `Account` (ie. `Archive this account`) for all of them?
